tools/workspace-plugin/src/executors/build/__fixtures__/executor/libs/proj/tsconfig.lib.json

```diff
@@ -2,7 +2,9 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "./dist/out-tsc",
-    "declaration": true
+    "declaration": true,
+    "types": []
   },
-  "include": ["src/*.ts"]
+  "include": ["src/*.ts"],
```

Comment:
this was consuming all node_modules @types and spec files which is not reflecting real environment and was also causing type errors ( which flawed the e2e test )